project.clj

```diff
@@ -1,19 +1,21 @@
-(defproject com.soundcloud/prometheus-clj "2.4.1-SNAPSHOT"
-  :description "Clojure wrappers for the Prometheus java client"
+(defproject ilovezoona/prometheus-clj "2.4.2-SNAPSHOT"
```

Comment:
Is the name change intentional? Not sure we want to merge that. Please also keep version bumps outside of PRs, they happen during new releases.